ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/admin/PackageManagementClient.scala

```diff
@@ -36,4 +37,12 @@ final class PackageManagementClient(service: PackageManagementServiceStub)(impli
       .uploadDarFile(UploadDarFileRequest(darFile))
       .map(_ => ())
 
+  def uploadDarFilePrime(
```

Comment:
Why is this needed? The response is empty, and the failure behaviour is propagated through the `.map` above

Reply:
Some experiment that didn't survive - removed.